### PR TITLE
Bash client to lsp-bash.el

### PR DESCRIPTION
**Pull request by elxbarbosa**
Hi,

minor changes:
- defgroup's `:group 'lsp-mode`. 
- make-lsp-client`s `server-id lsp-bash`

### CODE REVIEW

File: lsp-bash.el [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+(lsp-register-client
+ (make-lsp-client
+  :new-connection (lsp-stdio-connection #'lsp-bash--bash-ls-server-command)
+  :major-modes '(sh-mode)
+  :priority -1
+  :environment-fn (lambda ()
+                    '(("EXPLAINSHELL_ENDPOINT" . lsp-bash-explainshell-endpoint)
+                      ("HIGHLIGHT_PARSING_ERRORS" . lsp-bash-highlight-parsing-errors)
+                      ("GLOB_PATTERN" . lsp-bash-glob-pattern)))
+  :server-id 'lsp-bash
```
Comment (kiennq) [inline lsp-bash.el:74]: keep this as `bash-ls` is also fine too.

Comment (kiennq) [reply lsp-bash.el:74]: Else you will need to update doc too.

Comment (elxbarbosa) [reply lsp-bash.el]: done!

**Comment by yyoncho** [comment]
Can you resolve changes?

**Comment by elxbarbosa** [comment]
done.

Wow, Magit is just awesome 👍🏿

**Comment by ericdallo** [comment]
Thank you!

**Pull request merged**